packages/config/src/handlers/applicationCertificateHandler.ts

```diff
@@ -0,0 +1,150 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+import * as Hapi from '@hapi/hapi'
+import Certificate, {
+  IApplicationCertificateModel,
+  Status,
+  Event
+} from '@config/models/Certificate' //   IApplicationConfigurationModel
+import { logger } from '@config/config/logger'
+import * as Joi from 'joi'
+import { badRequest } from '@hapi/boom'
+import { isValidSVGCode } from '@config/services/applicationCertificateService'
+interface IActivePayload {
+  status: Status
+  event: Event
+}
+
+export async function getCertificateHandler(
+  request: Hapi.Request,
+  h: Hapi.ResponseToolkit
+) {
+  const { status, event } = request.payload as IActivePayload
+  const certificate: IApplicationCertificateModel | null =
+    await Certificate.findOne({ status: status, event: event })
+
+  return certificate
+}
+
+export async function getActiveCertificatesHandler(
+  request: Hapi.Request,
+  h: Hapi.ResponseToolkit
+) {
+  const activeCertificats: IApplicationCertificateModel | {} =
```

Comment:
What's the variable type if you leave the type completely undefined? From my understanding, Mongoose returns an `IApplicationCertificateModel | null` type if the value isn't found, but I might be wrong. I'd say let's only define types when TS compiler cannot infer them automatically

Reply:
Yeah, I will remove the defined type from this so that it will infer to its actual types.